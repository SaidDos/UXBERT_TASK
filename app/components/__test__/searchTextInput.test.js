import 'react-native';
import React from 'react';
import SearchTextInput from '../textInputs/searchTextInput';

import renderer from 'react-test-renderer';

it('should renders correctly', () => {
  const searchTerm = 'someSearchTerm';
  // eslint-disable-next-line no-alert
  const onchangeText = () => alert('someChangingText');
  const tree = renderer
    .create(
      <SearchTextInput onchangeText={onchangeText} searchTerm={searchTerm} />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
