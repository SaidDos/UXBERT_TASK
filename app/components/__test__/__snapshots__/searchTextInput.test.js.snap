// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Search by title..."
      placeholderTextColor="#800080"
      rejectResponderTermination={true}
      style={
        Object {
          "alignSelf": "center",
          "height": 40,
          "width": 300,
        }
      }
      underlineColorAndroid="transparent"
      value="someSearchTerm"
    />
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#800080",
            "fontSize": 30,
          },
          Object {
            "alignSelf": "center",
          },
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ï’¥
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "lightgrey",
          "height": 1,
          "width": "100%",
        },
        undefined,
      ]
    }
  />
</View>
`;
